contracts/src/systems/order/take_fungible_order.cairo

```diff
@@ -90,13 +91,25 @@ mod TakeFungibleOrder {
             let travel_time = caravan_position
                 .calculate_travel_time(taker_position, movable.sec_per_km);
 
+            let mut taker_order_travel_time: u64 = travel_time;  
```

Comment:
It's to prevent rounding errors. I have to ensure multiplication is done before division. https://soliditydeveloper.com/solidity-design-patterns-multiply-before-dividing

Reply:
makes sense